adaptive_shield/assets/dashboards/adaptive_shield_overview.json

```diff
@@ -0,0 +1 @@
+{"title":"Adaptive Shield Alerts Dashboard","description":"## Adaptive Shield Alerts Dashboard\n\nMonitor and analyze alerts from Adaptive Shield: configuration drifts, integration failures and security check degradations.\n","widgets":[{"id":5819790721138146,"definition":{"title":"","title_size":"16","title_align":"left","time":{},"type":"event_stream","query":"source:adaptive_shield","event_size":"s"},"layout":{"x":0,"y":0,"width":4,"height":4}},{"id":2633793451677842,"definition":{"title":"Alerts type distribution","title_size":"16","title_align":"left","time":{},"requests":[{"formulas":[{"formula":"query1","limit":{"order":"desc"}}],"style":{"palette":"classic"},"response_format":"scalar","queries":[{"data_source":"events","name":"query1","search":{"query":"source:adaptive_shield"},"indexes":["*"],"compute":{"aggregation":"cardinality","metric":"@evt.id"},"group_by":[{"facet":"type","limit":10,"sort":{"aggregation":"cardinality","order":"desc","metric":"@evt.id"}}]}]}],"type":"sunburst","legend":{"type":"automatic"}},"layout":{"x":4,"y":0,"width":4,"height":4}},{"id":5325858656995348,"definition":{"title":"Alerts distribution by Integration","title_size":"16","title_align":"left","time":{},"type":"query_table","requests":[{"formulas":[{"conditional_formats":[],"cell_display_mode":"bar","formula":"query1","limit":{"count":10,"order":"desc"}}],"response_format":"scalar","queries":[{"data_source":"events","name":"query1","search":{"query":"source:adaptive_shield"},"indexes":["*"],"compute":{"aggregation":"cardinality","metric":"@evt.id"},"group_by":[{"facet":"@aggregation_key","limit":10,"sort":{"aggregation":"cardinality","order":"desc","metric":"@evt.id"}}]}]}],"has_search_bar":"auto"},"layout":{"x":8,"y":0,"width":4,"height":4}},{"id":2809903660624346,"definition":{"title":"Alerts received (by type)","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"time":{},"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"data_source":"events","name":"query1","search":{"query":"source:adaptive_shield"},"indexes":["*"],"compute":{"aggregation":"cardinality","metric":"@evt.id"},"group_by":[{"facet":"type","limit":100,"sort":{"aggregation":"cardinality","order":"desc","metric":"@evt.id"}}]}],"response_format":"timeseries","style":{"palette":"red","line_type":"solid","line_width":"normal"},"display_type":"bars"}],"yaxis":{"include_zero":false,"scale":"log"}},"layout":{"x":0,"y":4,"width":12,"height":5}}],"template_variables":[],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"ceh-24y-jb3"}
```

Comment:
A few comments on the dashboard:

- Could you update this widget title to be more descriptive, i.e. maybe "Adaptive Shield Event Stream"?
    - <img width="535" alt="Screen Shot 2022-10-24 at 2 34 47 PM" src="https://user-images.githubusercontent.com/22504341/197600084-620812c4-9ebd-40b5-b578-38d7fbcbf859.png">
- We recommend adding a logo/branding onto the dashboard, as well as a summary of what the widgets represent.

Reply:
Yes, adding a summary to a text widget is fine, thanks!